package.json

```diff
@@ -20,7 +20,7 @@
   "dependencies": {
     "concat-stream": "1.6.2",
     "debug": "2.6.9",
-    "mkdirp": "0.5.1",
+    "mkdirp": "0.5.3",
```

Comment:
How about relaxing to allow further minor upgrades in the future too?

```suggestion
    "mkdirp": ">=0.5.3 <1",
```

Reply:
Although with `mkdirp` this will be less relevant, as it is now unsupported...

Maybe this should be applied for the other dependencies though.

Reply:
Latest version of mkdirp is `1.0.3`, and it seems supported

Reply:
Should I commit `^0.5.3` ?

Reply:
@SakiiCode there's actually no difference here, both match `0.5.x` where `x >= 3`

Reply:
Ok, added this suggestion